src/pages/PremiumVideoCall/App.test.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * This file was modified by
+ * Mattia Primavera <sconqua@gmail.com>
+ */
+
+import React from 'react'
+import App from './App'
+import MenuBar from './components/MenuBar/MenuBar'
+import PreJoinScreens from './components/PreJoinScreens/PreJoinScreens'
+import Room from './components/Room/Room'
+import { shallow } from 'enzyme'
+import useHeight from './hooks/useHeight/useHeight'
+import useRoomState from './hooks/useRoomState/useRoomState'
+
+jest.mock('./hooks/useRoomState/useRoomState')
+jest.mock('./hooks/useHeight/useHeight')
+
+const mockUseRoomState = useRoomState as jest.Mock<any>
+const mockUseHeight = useHeight as jest.Mock<any>
+
+mockUseHeight.mockImplementation(() => '500px')
+
+describe('the App component', () => {
+    it('should render correctly when disconnected from a room', () => {
```

Comment:
Enzyme or testing library ? Maybe it would be better to choose one testing lib. I know there're differents

Reply:
I think preniumVideoCall codes comes from Twilio directly. So it may be better to just not review this. 

@MattiaPrimavera , why do you think of not passing prettier on those files so it may be easier to update them later on base on the source?

Reply:
I thought the original example project would not evolve that much indeed, so I
- thought of using the original project only as a way to boost the initial development
- did not want to lose the *format on save* comfort
- privileged a consistent way of formatting throughout the whole project

In case we really want to profit of all changes from the original project, we may also consider declaring this as a **git submodule** and be able to `pull` new changes, instead of reporting those here. Still, we'll have to handle merges.

Reply:
yeah, not fan of submodule in this very specific case, maybe we could adapt prettier to match twillio prettier for those files. My point is that if someone want to update the code later on, at list a basic diff will do the job, painless but ok.